script.js

```diff
@@ -0,0 +1,165 @@
+// create library and constructor
+const library = [];
+const $bookList = document.querySelector('ul');
+const $addBookButton = document.querySelector('.add-book-button');
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+// add placeholder books
+function addPlaceholderBooks() {
+  const lordOfTheFlies = new Book('Lord of the flies', 'William Golding', 345, false);
+  const theHungerGames = new Book('The hunger games', 'Suzanne Collins', 400, false);
+  library.push(lordOfTheFlies, theHungerGames);
+}
+
+
+function showBook(title, author, pages, read) {
+  const li = document.createElement('li');
+  const bookName = document.createElement('span');
+  const titleLabel = document.createElement('span');
+  const bookAuthor = document.createElement('span');
+  const authorLabel = document.createElement('span');
+  const bookPages = document.createElement('span');
+  const pagesLabel = document.createElement('span');
+  const bookRead = document.createElement('span');
+  const readLabel = document.createElement('span');
+  const deleteBtn = document.createElement('span');
+
+  // changing the style and content of the elements
+  bookName.textContent = title;
+  bookName.classList.add('name');
+  titleLabel.classList.add('label', 'title-label');
+  titleLabel.textContent = 'Title:';
+  bookAuthor.textContent = author;
+  bookAuthor.classList.add('author');
+  authorLabel.classList.add('label', 'author-label');
+  authorLabel.textContent = 'Author:';
+  bookPages.textContent = pages;
+  bookPages.classList.add('pages');
+  pagesLabel.classList.add('label', 'pages-label');
+  pagesLabel.textContent = 'Number of Pages:';
+  bookRead.textContent = read;
+  bookRead.classList.add('read');
+  readLabel.classList.add('label', 'read-label');
+  readLabel.textContent = 'Read?:';
+  deleteBtn.textContent = 'delete';
+  deleteBtn.classList.add('delete');
+
+  li.appendChild(titleLabel);
+  li.appendChild(bookName);
+  li.appendChild(authorLabel);
+  li.appendChild(bookAuthor);
+  li.appendChild(pagesLabel);
+  li.appendChild(bookPages);
+  li.appendChild(readLabel);
+  li.appendChild(bookRead);
+  li.appendChild(deleteBtn);
+  $bookList.appendChild(li);
+}
+function switchBack() {
+  const hidden = document.querySelector('.hidden-form');
+  hidden.classList.add('hidden');
+  const wrapper = document.querySelector('#wrapper');
+  wrapper.classList.remove('hidden');
+}
```

Comment:
### Separate dom manipulation from app logic

Reply:
### Hi there, regarding the previous TSE's requirements:
---
- we changed all of the functions to arrow functions.
- the previous TSE asked us to configure and correct our linters errors, we already did that.
- he requested from us that we should separate the DOM manipulation syntax, but no function has both together.
---

If you find any other issues, please be more clear about where the issues are.